src/WidgetBase.ts

```diff
@@ -198,13 +191,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		this._properties = <P> {};
```

Comment:
Would `Object.create(null)` perform better?  I don't if this gets overwritten or not, but if it doesn't, I suspect it will be a minor performance improvement in property lookups.